spec.go

```diff
@@ -1,9 +1,5 @@
 package certificatetpr
 
-type Searcher interface {
-	SearchCerts(clusterID string) (AssetsBundle, error)
-}
-
```

Comment:
Why does this change? IMO it was correct before.

Reply:
Reverted, thx @xh3b4sd!